azure/azure-explanation/security-overview.rst

```diff
@@ -0,0 +1,109 @@
+Ubuntu Security on Azure
+========================
+
+Ubuntu security features
+------------------------
+
+Ubuntu on Azure inherits and benefits from all of the security features available to Ubuntu Server. Learn more about
+Ubuntu Server security features and security-related topics with `Introduction to Security <https://documentation.ubuntu.com/server/explanation/intro-to/security/>`_.
+
+Azure security features
+-----------------------
+
+By default, all Ubuntu X64 Hyper-V Gen2 images on Azure support `Trusted Launch for Azure Virtual Machines <https://learn.microsoft.com/en-us/azure/virtual-machines/trusted-launch>`_.
+Ubuntu also produces images which support `Azure Confidential Virtual Machines <https://learn.microsoft.com/en-us/azure/confidential-computing/confidential-vm-overview>`_.
+
+See :doc:`confidential-computing`.
+
+Choosing the right image
+------------------------
+
+Ubuntu offers a range of image types on Azure to meet security requirements. The security features and capabilities
+of each image can be further enhanced with `Ubuntu Pro`_.
+
+Server
+~~~~~~
+
+Optimised for general server use cases, Ubuntu Server is ideal for users seeking a stable and secure OS foundation to
+support server workloads.
+
+Minimal
+~~~~~~~
+
+By providing a greatly reduced package set to minimise the security cross-section of the image, Ubuntu Minimal requires
+fewer security updates over time.
+
+CIS
+~~~
+
+A hardened Ubuntu image (see `Image Hardening`_), pre-configured to align with industry best practices and guidelines
+set by the Center for Internet Security (CIS). Ubuntu CIS images on Azure use custom tailoring files to meet both CIS
+benchmarks as well as ensure the image remains functional in Azure cloud environments.
+
+FIPS
+~~~~
+
+The Ubuntu FIPS image contains a Ubuntu kernel and security components which have been certified by the National
+Institute of Standards and Technology (NIST), making it a compelling choice for users operating in regulated
+environments such as FedRAMP, HIPAA, or PCI.
```

Comment:
Check with Henry that FIPS helps with HIPPAA or PCI. It may be that CIS is more appropriate.

Reply:
HIPAA and PCI are lifted directly from other Canonical FIPS documentation, e.g. AWS FIPS https://ubuntu.com/aws/fips